gradle/android_dependencies.gradle

```diff
@@ -25,7 +25,11 @@ dependencies {
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$gradle.kotlinx_coroutines_version"
     testImplementation 'org.jetbrains.kotlin:kotlin-test'
     testImplementation 'org.jetbrains.kotlin:kotlin-test-junit'
+    testImplementation "org.junit.jupiter:junit-jupiter-api:$gradle.android_test_junit_jupiter_version"
 
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$gradle.android_test_junit_jupiter_version"
+
```

Comment:
Does something actually need this?

Reply:
Yes. I don't have that much experience with Android so I just copypasted from another module, but I was not able to run android (not instrumented) unit tests without it. idea doesn't just sais "No tests are found"

Reply:
This is for your own tests? (as you can see on CI all the other modules have no problem running). 

Android tests are JUnit4 based by default, this seems JUnit5 related.

Reply:
Just added some tests and idea doesn't see them. Don't know why, but okey, I'll remove it

Reply:
give me the branch and I can help figure out why.

Reply:
There is no branch, I just add some dummy sources and tests to this project to check that something that we recommend to use as a template works. So you can clone this branch and add some simple class ( class Dummy(val description: String) android unit tests following instructions in README. In my case, IDE cannot find Android unit tests. Of course, it could be some problem on my local machine, so I just removed it form android_dependencies.

Reply:
If adding this fixes it for you, there's probably still something wrong with the way you wrote your test, or something is broken. 

I noticed android integration tests are not on by default on Bitrise yet, I'll enable them.